sale_stock_available_info_popup/static/src/xml/qty_at_date.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<templates>
+    <div t-extend="sale_stock.QtyDetailPopOver">
+        <t t-jquery="t[t-if*='!data.is_mto']" t-operation="append">
+            <tr>
+                <td><strong>Available to promise</strong></td>
+                <td><t t-esc='data.immediately_usable_qty_today'/>
+                <t t-esc='data.product_uom.data.display_name'/></td>
```

Comment:
I can't test it... but is this correct?

Reply:
Yes @Tardo It is in this way, because of the data structure in 'data'. Thanks.